utils/constants.py

```diff
@@ -41,3 +41,56 @@
 name_input_value = "input_value"
 name_processing_value = "processing_value"
 name_understanding_value = "understanding_value"
+
+# the following are the LE formats used in RGB
+rgb_le_variables = [
+    "CT",
+    "BO",
+    "LG",
+    "MS",
+    "QU",
+    "EX",
+    "SU",
+    "AAM",
+    "VAM",
+    "TAM",
+]
+
+# this dict maps output variable states of BN
+# to RGB LE formats
+le_name_map = {
+    "{'CT': 'Yes'}": "kollaborativ",
+    "{'BO': 'Yes'}": "kurzuebersicht",
+    "{'LG': 'Yes'}": "lernziele",
```

Comment:
You can exchange the strings with the constants from above.
```suggestion
    "{'BO': 'Yes'}": name_kü,
```